test/specs/test.e2e.ts

```diff
@@ -30,6 +30,18 @@ describe('Marquee', () => {
       await expect(webview.widgets$$).toBeElementsArrayOfSize(8)
     })
 
+    describe('fullscreen feature', () => {
+      for (const widget of WIDGETS) {
+        it(`${widget} widget`, async () => {
+          const modalSelector = `.MuiModal-root div[aria-labelledby="${widget}Fullscreen"]`
+          await $(`button[aria-label="Toggle ${widget} widget to fullscreen"]`).click()
+          await expect($(modalSelector)).toBeExisting()
+          await browser.keys(['Escape'])
+          await expect($(modalSelector)).not.toBeExisting()
+        })
+      }
+    })
+
```

Comment:
This can be removed, probably an artifact from the rebase (you find the same code already in the bottom of the file)
```suggestion
```